src/pretalx/cfp/templates/cfp/event/user_submission_invitation.html

```diff
@@ -0,0 +1,35 @@
+{% extends "cfp/event/base.html" %}
+{% load i18n %}
+{% load bootstrap4 %}
+{% block title %}{% trans "Your submission" %} :: {{ request.event }}{% endblock %}
+
+{% block content %}
+    <h2>
+        <a href="{{ submission.urls.user_base }}">
+            {% trans "Your submission:" %} {{ submission.title }}
+        </a>
+    </h2>
+    <p>
+        {% trans "Current state of your submission:" %}
+        {% include "cfp/event/fragment_state.html" with state=submission.state %}
+    </p>
+    <div class="alert alert-info">
+        {% blocktrans trimmed %}
+            Invite another speaker to your submission here. Instead, you can also send them
+            this link:
+        {% endblocktrans %}
+        <br>
+        {{ invite_url }}
```

Comment:
Maybe make this link clickable, so users can do RightClick=>CopyLink?

Reply:
I normally accidentally click links like that instead. In reality we'd need a "click to copy" link, but I'd open that as an easy issue instead.